.github/workflows/deploy-worker.yml

```diff
@@ -115,3 +122,23 @@ jobs:
           wranglerVersion: ${{ env.wranglerVersion }}
           workingDirectory: ${{ github.event.inputs.directory }}
           command: deploy --no-bundle --name=${{ github.event.inputs.appId }} --dispatch-namespace ${{ github.event.inputs.dispatchNamespace }} ${{ steps.get-script.outputs.WORKER_SCRIPT }}
+
+  attest:
+    needs: deploy
+    runs-on: ubuntu-latest
+
+    permissions:
+      id-token: write
+      attestations: write
+
+    steps:
+      - name: Download worker script
+        uses: actions/download-artifact@v4
+        with:
+          name: worker_script
+          path: ./
+
+      - name: Attest Build Provenance
+        uses: actions/attest-build-provenance@v1
+        with:
+          subject-path: ${{ needs.deploy.outputs.WORKER_SCRIPT }}
```

Comment:
Upload bundle (& script/artifact) to r2?
https://github.com/actions/attest-build-provenance?tab=readme-ov-file#outputs

Still not sure how to get the attestation url outside of via the the script/artifact digest
https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#list-attestations
https://github.com/actions/attest-build-provenance/issues/69#issuecomment-2106382628